components/admin/users/AdminUsersTable.tsx

```diff
@@ -56,9 +57,9 @@ const AdminOption: React.FC<AdminOptionProps> = ({
     }
   }
 
-  const changeButton = async () => {
-    await changeRights(mutationVariable)
```

Comment:
There is no reason to wait for this mutation since its return value isn't used anywhere.